File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/ArbeidsforholdEtterUforgrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import java.io.Serializable;
import java.util.Date;

/**
 * @author Lars Hartviksen (Decisive), PK-6985
 * @author Aasmund Nordstoga (Accenture) - PK-16378, PKPYTON-1754
 */
public class ArbeidsforholdEtterUforgrunnlag implements Serializable {


    /**
     * Fom date for arbeidsforholdet.
     */
    private Date fomDato;

    /**
     * Work load.
     */
    private int stillingsprosent;

    /**
     * if it is lasting facilitated work
     */
    private boolean varigTilrettelagtArbeid;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/ArbeidsforholdEtterUforgrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import java.io.Serializable
import java.util.*
import java.time.LocalDate

class ArbeidsforholdEtterUforgrunnlag : Serializable {
    /**
     * Fom date for arbeidsforholdet.
     */
    var fomDatoLd: LocalDate? = null

    /**
     * Work load.
     */
    var stillingsprosent = 0

    /**
     * if it is lasting facilitated work
     */
    var varigTilrettelagtArbeid = false

}